ivy/functional/frontends/jax/numpy/statistical.py

```diff
@@ -427,3 +427,17 @@ def ptp(a, axis=None, out=None, keepdims=False):
     x = ivy.max(a, axis=axis, keepdims=keepdims)
     y = ivy.min(a, axis=axis, keepdims=keepdims)
     return ivy.subtract(x, y)
+
+
+@handle_jax_dtype
+@to_ivy_arrays_and_back
+def nanmean(a, axis=None, dtype=None, out=None, keepdims=False, *, where=None):
+    axis = tuple(axis) if isinstance(axis, list) else axis
```

Comment:
as per [docs](https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.nanmean.html), the default dtype is float64. I would use the link here to just check that all behavior is the same :)

Reply:
I used float64 as stated in the docs